tgui/packages/tgui/interfaces/AresInterface.jsx

```diff
@@ -279,6 +280,19 @@ const MainMenu = (props) => {
                 onClick={() => act('page_access')}
               />
             </Stack.Item>
+            <Stack.Item>
+              <Button
+                content="Core Security Protocols"
+                tooltip="Utilise the AI Core security features."
+                icon="shield"
+                color="red"
+                ml="auto"
+                px="2rem"
+                width="27vw"
```

Comment:
You should standardize all buttons to be a particular width rather than have some be larger than others.